SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -133,20 +133,64 @@ CEulerSolver::CEulerSolver(void) : CSolver() {
   ExtAverageOmega            = NULL;
 
 
-  /*--- Initialize primitive quantities for turboperformace ---*/
-
-  DensityIn                     = NULL;
-  PressureIn                    = NULL;
-  TurboVelocityIn               = NULL;
-  DensityOut                    = NULL;
-  PressureOut                   = NULL;
-  TurboVelocityOut              = NULL;
-  KineIn                        = NULL;
-  OmegaIn                       = NULL;
-  NuIn                          = NULL;
-  KineOut                       = NULL;
-  OmegaOut                      = NULL;
-  NuOut                         = NULL;
+  /*--- Initialize quantities for turboperformace ---*/
```

Comment:
I am wondering, I like you replaced `NULL` to `nullptr` but this looks quite inconsistent with general style of the code. I would open a separate issue to replace all `NULL` into `nullptr`